assets/sass/modules/_qtip.scss

```diff
@@ -36,3 +36,50 @@
     }
   }
 }
+
+/*! Custom qtip styles, based on dark theme */
+.qtip-custom {
```

Comment:
Looks like it's okay (passed the linter) - for this scss change I mostly just copied our overrides from before from inside the vendor css file, to our actual override file.